cranelift/codegen/src/machinst/abi.rs

```diff
@@ -830,7 +705,6 @@ pub struct SigSet {
     /// The actual ABI signatures, keyed by `Sig`.
     sigs: PrimaryMap<Sig, SigData>,
 }
-
```

Comment:
I wouldn't bother to mention this except I'm making some other requests already: could you put this blank line back?